From 7af28978a9711dff04d46e4780a0ab439a954ea0 Mon Sep 17 00:00:00 2001
From: Ayush Singh <ayushdevel1325@gmail.com>
Date: Wed, 23 Aug 2023 20:20:30 +0530
Subject: [PATCH 1/3] dt-bindings: Add beagleplaygreybus to devicetree

The UART is used for communication between gb-beagleplay driver and
BeaglePlay CC1352 running Greybus SVC application.

Signed-off-by: Ayush Singh <ayushdevel1325@gmail.com>
---
 arch/arm64/boot/dts/ti/k3-am625-beagleplay.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/ti/k3-am625-beagleplay.dts b/arch/arm64/boot/dts/ti/k3-am625-beagleplay.dts
index 7cfdf562b53b..50058ea9c1db 100644
--- a/arch/arm64/boot/dts/ti/k3-am625-beagleplay.dts
+++ b/arch/arm64/boot/dts/ti/k3-am625-beagleplay.dts
@@ -870,6 +870,10 @@ &main_uart6 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&wifi_debug_uart_pins_default>;
 	status = "okay";
+
+	beagleplaygreybus {
+		compatible = "beagle,beagleplaygreybus";
+	};
 };
 
 &dss {
-- 
2.39.2

